python/test/t_HODLRMatrix_std.py

```
@@ -0,0 +1,599 @@
+assert hodlr_off24.getPermutation().getSize() == 0, (
+    "flag OFF should disable the permutation"
+)
+# scrambled 2D grid with a short correlation length: the spatial ordering
+# preserves the accuracy and compresses the tree, while the blind split of
+# the unordered vertices falls back to dense storage
+im24 = ot.IntervalMesher([15, 15])
+mesh24 = im24.build(ot.Interval([0.0, 0.0], [1.0, 1.0]))
+vertices24 = mesh24.getVertices()
+n24 = vertices24.getSize()
+order2d24 = list(range(n24))
+random.Random(1).shuffle(order2d24)
```

Ruff (0.16.1)

[error] Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)